package.json

```diff
@@ -4,6 +4,18 @@
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
+  "exports": {
+    ".": {
+      "default": "./lib/index.js"
+    },
+    "./native": {
+      "default": "./lib/native/index.js"
+    },
+    "./node": {
+      "require": "./lib/node/index.js",
+      "default": "./lib/node/index.mjs"
```

Comment:
Does the "default" behavior mean ESM?

Reply:
Default would be anything not explicitly matched above - so `import` usage would fall into that export

Reply:
Got it. Thanks for the explanation!